DayZLifeServer/scripts/4_World/EventListener/DZLLoadOutListener.c

```diff
@@ -14,8 +20,15 @@ class DZLLoadOutListener: DZLBaseEventListener {
                     return;
                 }
 
+                if (IsCooldownActive(player)) {
+                    DZLSendMessage(sender, "You need to wait one hour to pickup Loadout again");
```

Comment:
❌ This will not translated. Pls use a placeholder like #YouNeedToWaitOneHour and add it to stringtable.csv